src/operator/nn/pooling-inl.h

```diff
@@ -81,7 +82,10 @@ struct PoolingParam : public dmlc::Parameter<PoolingParam> {
     .describe("Pad for pooling: (y, x) or (d, y, x). Defaults to no padding.");
 
     DMLC_DECLARE_FIELD(p_value).set_default(dmlc::optional<int>())
-    .describe("Value of p for Lp pooling, can be 1 or 2, required for Lp Pooling");
+    .describe("Value of p for Lp pooling, can be 1 or 2, required for Lp Pooling.");
+
+    DMLC_DECLARE_FIELD(count_include_pad).set_default(dmlc::optional<bool>())
+    .describe("Whether to count padding elements for average calculation.");
```

Comment:
Be more descriptive.
Only used for average pooling. Whether to count padded elements for normalization at edge and corners.
For example, blah blah